test/testdata/rewriter/attr_abstract.rb

```diff
@@ -0,0 +1,15 @@
+# typed: strict
+
+module IFoo
+  extend T::Sig, T::Helpers
+  abstract!
+
+  sig { abstract.returns(String) }
```

Comment:
Should we add some runtime tests to make sure that the sig is actually intercepting these?